src/CBOR.h

```diff
@@ -0,0 +1,65 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include <cstdint>
+#include <string>
+#include <vector>
+
+namespace TW {
+
+// Light CBOR writer implementation.
+class cbor_writer {
```

Comment:
```suggestion
class CBORWriter {
```
for class and function name, we perfer camel case

Reply:
Going to remove this CBOR implementation in favor of the new one